tests/cpu/ops/fixedfunction/FixedFunctionOpData_tests.cpp

```diff
@@ -155,4 +253,15 @@ OCIO_ADD_TEST(FixedFunctionOpData, is_inverse)
     OCIO_CHECK_ASSERT(!f_r->isInverse(f_r));
     OCIO_CHECK_ASSERT(!f_r_inv->isInverse(f_r_inv));
     OCIO_CHECK_ASSERT(!f_r->isInverse(f_g));
+
+    OCIO::FixedFunctionOpData::Params p7 = { 1.147, 1.264, 1.312, 0.815, 0.803, 0.880, 1.2 };
+    auto f_gm = std::make_shared<const OCIO::FixedFunctionOpData>(
+        OCIO::FixedFunctionOpData::ACES_GAMUTMAP_13_FWD, p7);
+    auto f_gm_inv = std::make_shared<const OCIO::FixedFunctionOpData>(
+        OCIO::FixedFunctionOpData::ACES_GAMUTMAP_13_INV, p7);
+    OCIO_CHECK_ASSERT(f_gm->isInverse(f_gm_inv));
+    OCIO_CHECK_ASSERT(f_gm_inv->isInverse(f_gm));
+    OCIO_CHECK_ASSERT(!f_gm->isInverse(f_gm));
+    OCIO_CHECK_ASSERT(!f_gm_inv->isInverse(f_gm_inv));
+    OCIO_CHECK_ASSERT(!f_gm->isInverse(f_r));
```

Comment:
Would be nice to change f_gm_inv pwr to 1.21 and verify that it is no longer an inverse of f_gm.

Reply:
Added as suggested.